packages/client/src/services/posts.ts

```diff
@@ -21,6 +21,19 @@ const getAll = async () => {
   return response;
 };
 
+const getPostsByFilter = async (postState: PostState) => {
+  const response = await axios.get(`${baseUrl}`, {
+    params: {
+      posts: [],
+      sortType: postState.sortType,
+      locationFilter: postState.locationFilter,
```

Comment:
Not sure how I feel about objects in query params LOL. We should prob just separate lat and lon